### PR TITLE
Imp notification address

### PR DESCRIPTION
**Pull request by marta197**
### Case
A Client can have multiple Projects. That Client now have just one address, but wants to have a different address for a specific project.

### Proposed solution
Create a `NotificationAddress` model associated with the Client and have that `notification_address` in the Project (with the client notification information for that project).

### CODE REVIEW

File: somsolet/models/client.py [hunk 1]
```diff
@@ -48,6 +48,20 @@ class Client(models.Model):
         verbose_name=_('DNI'),
     )
 
+    def __str__(self):
+        return self.name
+
+
+class NotificationAddress(models.Model):
```
Comment (juan-pe) [inline somsolet/models/client.py:55]: What do you think if we add the arguments `related_name` and `related_query_name` to get client notification address more easily?

ref: https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.ForeignKey.related_name

**Review by juan-pe — APPROVED** [review]
Wow! this is a super nice refactor!

only the comment about if you want to improve the relationship between client and notification address models

thanks! :tophat: :womans_hat: :star2:

File: somsolet/models/client.py (hunk 1 above)
Comment (marta197) [reply somsolet/models/client.py:86]: Hello! :wave: 
We've added `related_name` in `client_file` and `notification_address` ForeignKeys in Project.
Is this what you meant?

**Review by juan-pe — APPROVED** [review]
yuhuuuuu! :tada: :confetti_ball:

**Pull request merged**